src/raster/r.mess/r.mess.py

```diff
@@ -192,9 +193,19 @@ def main(options, flags):
         return 0
 
     # Reference / sample area or points
-    ref_rast = options["ref_rast"]
     ref_vect = options["ref_vect"]
-    if ref_rast:
+    if bool(ref_vect):
```

Comment:
I don't think the bool conversion is needed, it's automatic.

Reply:
OK, thanks. It was recommended to me once, but can't remember the specifics.

Reply:
Done